product_docs/docs/biganimal/release/free_trial/detail/create_an_account/index.mdx

```diff
@@ -4,17 +4,18 @@ description: "Creating an EDB account and signing into the BigAnimal free trial
 navTitle: "Create an account" 
 ---
 
-Registering an account on [enterprisedb.com](https://www.enterprisedb.com/) gets you access to the BigAnimal trial, as well as EDB's training and software downloads. 
+You will need to register with EDB in order to access BigAnimal free trial, as well as EDB's training and software downloads.
```

Comment:
You need to register with EDB to access the BigAnimal free trial and EDB's training and software downloads.

(no future tense, "in order to" is wordy)